.github/workflows/python.yml

```diff
@@ -10,19 +10,18 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v4
-    # NB: v1.4.0 covers Python 3.8.
-    - uses: ricardochaves/python-lint@v1.4.0
+    - uses: actions/setup-python@v5
       with:
-        python-root-list: lddtree.py pylint
```

Comment:
it's linting 2 files here, not only lddtre.py

Reply:
What does the "pylint" file actually do? As far as I can tell it doesn't do anything that running pylint directly wouldn't do?

Reply:
it is somewhat of a boilerplate script that is more useful in larger projects, but it makes it so you can run `./pylint` and it auto-discovers all the relevant files.  `pylint` exits telling you to specify the files to lint.

really the issue isn't the local pylint script, it's making sure the action can scale up beyond a single file.  i really need to add unittest coverage to lddtree.py for example.